app/src/main/java/com/tristanwiley/chatse/chat/ChatActivity.kt

```diff
@@ -450,6 +456,20 @@ class ChatActivity : AppCompatActivity(), ServiceConnection {
         Log.d("ChatActivity", "Service disconnect")
     }
 
+    override fun onItemClick(chatRoom: ChatRoom) {
+
+        val roomNo = chatRoom.num
```

Comment:
Try not to mix snake_case with camelCase on the same project. I'd rather only use camel case.